src/sys.rs

```diff
@@ -27,7 +26,7 @@ pub struct SystemProperties {
 
 pub fn get_info() -> SystemProperties {
     let info = os_info::get();
-    let locale = locale().unwrap_or_default();
+    let locale = sys_locale::get_locale().unwrap_or(String::from("unavailable")); // locale() not available YET in tauri v2
```

Comment:
can we keep unwrap_or_default here as well?

Reply:
yes